scripts/README.md

```diff
@@ -4,17 +4,22 @@ Below are the instructions to setup ROS development environment
 ## Setup
 
 1. Update /etc/hosts
-
 ```
 127.0.0.1       kafka
 127.0.0.1       minio
 ```
+
 2. Create require directories for minio.
 ```
 mkdir -p /usr/share/minio/{minio-conf,minio-data}
 ```
 
-3. Clone ros-backend repository and start necessary services in container.
+3. To make changes to ROS configurations, copy the example file and rename it as `.env` and update the environment variables.
+```
+cd ros; cp .env.example .env
+```
+
```

Comment:
I think this should go in the main README file, as this step will be applicable in production too. 
Probably it can live in both the README files.

Reply:
Sure, makes sense to have it in the main README.md